pkg/catalog/helm/setup.go

```diff
@@ -193,6 +193,10 @@ func (h *Helm) updateGit(fetchLatest bool) (string, error) {
 		}
 		if changed {
 			if err = git.Update(h.LocalPath, commit); err != nil {
+				if strings.Contains(err.Error(), h.password) {
+					errStr := strings.Replace(err.Error(), h.password, "****", 1)
```

Comment:
I think we should follow the convention from go [here](https://golang.org/src/net/http/client.go?s=30045:30300#L928) so more like `strings.Replace(err.Error(), h.password+"@", "***@", 1)`